src/api/users/mutations/loginUser/index.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import bcrypt from 'bcrypt'
-import type { ValidationError } from '@types/ValidationError'
+import { type ValidationError } from '@types/ValidationError'
```

Comment:
What we did before the change apparently works but the VSCode plugin would show it as `any`.
At least here it works for both Flow and the plugin.